src/pages/css/css-cursors/index.md

```diff
@@ -3,13 +3,52 @@ title: CSS Cursors
 ---
 ## CSS Cursors
 
-This is a stub. <a href='https://github.com/freecodecamp/guides/tree/master/src/pages/css/css-cursors/index.md' target='_blank' rel='nofollow'>Help our community expand it</a>.
+<!--This is a stub. <a href='https://github.com/freecodecamp/guides/tree/master/src/pages/css/css-cursors/index.md' target='_blank' rel='nofollow'>Help our community expand it</a>.
 
-<a href='https://github.com/freecodecamp/guides/blob/master/README.md' target='_blank' rel='nofollow'>This quick style guide will help ensure your pull request gets accepted</a>.
+<a href='https://github.com/freecodecamp/guides/blob/master/README.md' target='_blank' rel='nofollow'>This quick style guide will help ensure your pull request gets accepted</a>.-->
 
 <!-- The article goes here, in GitHub-flavored Markdown. Feel free to add YouTube videos, images, and CodePen/JSBin embeds  -->
+The cursor property specifies the type of cursor to be displayed when you hover over an element. It has 36 possible values: 
+
+    .auto            { cursor: auto; }
```

Comment:
Can you remove all indenting and enclose these in a code block using back ticks

The guide will automatically add syntax highlighting if you start the code block with ` ```css`